scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -40,7 +40,19 @@ RUN set -ex; \
 		libcvc4-dev libz3-static-dev z3-static jq \
 		; \
 	apt-get install -qy python3-pip python3-sphinx; \
-	pip3 install codecov; \
+	pip3 install codecov;
+
+# Eldarica
+RUN set -ex; \
+	apt-get update; \
+	apt-get install -qy unzip default-jre; \
```

Comment:
One thing to note - `default-jre` on Ubuntu 22.04 will install Java11, whereas on OSX, it'll install Java21. Given that the build is passing, it doesn't seem to be a problem - but just wanted to point out in case you think these need to be synced. Not sure what's the minimum required version for Eldarica.

Reply:
Yeah, it looks like JRE is not available as standalone on MacOS since Java11. And yeah, I believe you can install it just like you said - `brew install openjdk@11`.